sys/net/gnrc/netif/init_devs/auto_init_sx127x.c

```diff
@@ -61,7 +61,7 @@ void auto_init_sx127x(void)
 #endif
 
         sx127x_setup(&sx127x_devs[i], &sx127x_params[i]);
-#ifdef MODULE_GNRC_LORAWAN
+#ifdef MODULE_GNRC_NETIF_LORAWAN
```

Comment:
We could change this into a C conditional if we remove the `#ifdef` from the header inclusion. I think that would look cleaner.

Reply:
Done